src/clj/rems/util.clj

```diff
@@ -60,6 +60,15 @@
   ([user]
    (get user "mail")))
 
+(defn deep-merge [a & maps]
+  (if (map? a)
```

Comment:
I'm not sure why this case is needed. Why not just
```
(defn deep-merge [& maps]
  (apply merge-with deep-merge maps))

Reply:
It is needed. Without it the following happens:

(deep-merge {:a {:b {:c 1 :d 2}}} {:a {:b {:c 100}}})

expected: {:a {:b {:c 100, :d 2}}}
  actual: ({:a {:b {:c 100}}})